### ISSUE TITLE
More trouble with dots

### ISSUE THREAD

**Issue by Maltysen** [cheddar-lang/Cheddar#35]
`6` gives `6`
`6.` gives `6`
`6..` gives `6`
`6...` gives `NaN`?!??!

**Comment by vihanb** [comment]
This is because:

```
6  // number
6. // number with empty decimal
6.. // number with no property e.g. 6..toString() to access the tostrign property
6... // access invalid property which borks everything
```

**Comment by vihanb** [comment]
Fixed in ebaa5455a4340ee46c169dd0e2e1e900664ad825
